Sources/SessionManager/SessionManager.swift

```diff
@@ -85,7 +81,7 @@ public class SessionManager {
             case let .success(data):
                 let msgDict = try JSONSerialization.jsonObject(with: data)
                 os_log("create session response is: %@", log: getTorusLogger(log: Web3AuthLogger.network, type: .info), type: .info, "\(msgDict)")
-                return sessionID
+                return sessionId
```

Comment:
Should we not also assign to self.sessionId here as well as return the sessionId?

Reply:
No shouldn't as in sfa sdk sessionId should be saved after createSession success response and after successful decoding in setResultUrl() in web3Auth sdk. Shouldn't play with local variable state.

Reply:
Ok, thanks for clarifying.